src/AppBundle/Service/Intuition.php

```diff
@@ -40,6 +40,10 @@ public static function serviceFactory(RequestStack $requestStack, SessionInterfa
         $intuition = new static('grantmetrics');
         $intuition->registerDomain('grantmetrics', $rootDir.'/../i18n');
         $intuition->setLang(strtolower($useLang));
+
+        // Also add US English, so we can access the locale information (e.g. for date formatting).
+        $intuition->addAvailableLang('en-us', 'US English');
```

Comment:
I would probably call it `'English (US)'` or is there some standard I'm unaware of? Language first would make it easier to find in the language dropdown.